packages/lit-dev-content/src/icons/check-code-icon.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {html} from 'lit';
+
+// Source: google fonts material icons
+export const checkCodeIcon = html`<svg
+  xmlns="http://www.w3.org/2000/svg"
+  enable-background="new 0 0 24 24"
+  height="24px"
+  viewBox="0 0 24 24"
+  width="24px"
+  fill="currentcolor"
+>
+  <g>
+    <rect fill="none" height="24" width="24" />
+  </g>
+  <g>
+    <path
+      d="M14.05,17.58l-0.01,0.01l-2.4-2.4l1.06-1.06l1.35,1.35L16.54,13l1.06,1.06 l-3.54,3.54L14.05,17.58z M12,2C6.5,2,2,6.5,2,12s4.5,10,10,10s10-4.5,10-10S17.5,2,12,2z M7.34,6.28l1.41,1.41l1.41-1.41 l1.06,1.06L9.81,8.75l1.41,1.41l-1.06,1.06L8.75,9.81l-1.41,1.41l-1.06-1.06l1.41-1.41L6.28,7.34L7.34,6.28z M12,20 c-2.2,0-4.2-0.9-5.7-2.3L17.7,6.3C19.1,7.8,20,9.8,20,12C20,16.4,16.4,20,12,20z"
+      fill-rule="evenodd"
+    />
+  </g>
```

Comment:
- xmlns is unnecessary on `<svg>` elements. 
- `enable-background` doesn't seem to do anything here (I removed it on the preview for the check-code icon, and there was no change).
- There is an empty `<rect>`.
- There are unneccessary `<g>`s.

```suggestion
// Source: google fonts material icons
export const checkCodeIcon = html`<svg
  height="24px"
  viewBox="0 0 24 24"
  width="24px"
  fill="currentcolor"
>
    <path
      d="M14.05,17.58l-0.01,0.01l-2.4-2.4l1.06-1.06l1.35,1.35L16.54,13l1.06,1.06 l-3.54,3.54L14.05,17.58z M12,2C6.5,2,2,6.5,2,12s4.5,10,10,10s10-4.5,10-10S17.5,2,12,2z M7.34,6.28l1.41,1.41l1.41-1.41 l1.06,1.06L9.81,8.75l1.41,1.41l-1.06,1.06L8.75,9.81l-1.41,1.41l-1.06-1.06l1.41-1.41L6.28,7.34L7.34,6.28z M12,20 c-2.2,0-4.2-0.9-5.7-2.3L17.7,6.3C19.1,7.8,20,9.8,20,12C20,16.4,16.4,20,12,20z"
      fill-rule="evenodd"
    />
```